spark/src/main/java/org/apache/zeppelin/spark/ZeppelinContext.java

```diff
@@ -364,6 +376,31 @@ public void runNote(String noteId, InterpreterContext context) {
     }
   }
 
+  /**
+   * get job status by zeppelin note id and paragraph id
+   * @param noteId
+   * @param paragraphId
+   * @return
+   */
+  @ZeppelinApi
+  public RemoteZeppelinJobStatus getZeppelinJobStatus(String noteId, String paragraphId) {
```

Comment:
How about remove 'Zeppelin' from method name? so using this api will looks like

before
```
z.getZeppelinJobStatus()
```
after
```
z.getJobStatus()
```

So we API becomes more consistent along with other existing APIs, they don't include 'Zeppelin' in method name, i.e.

```
z.run()
z.runNote()
z.show()
z.getJobStatus()
```